BridgeAppSDK/SBAUserWrapper+Bridge.swift

```diff
@@ -517,7 +517,14 @@ public extension SBAUserWrapper {
         }
     }
     
-    fileprivate func updateFromUserSession(_ response: SBAUserSessionInfoWrapper) {
+    /**
+     Call this any time the user session info gets updated.
+     */
+    public func updateFromUserSessionInfo(_ info: SBBUserSessionInfo) {
+        self.updateFromUserSession(info)
+    }
+    
+    private func updateFromUserSession(_ response: SBAUserSessionInfoWrapper) {
```

Comment:
well it technically doesn't need to be public I suppose, but at least not fileprivate so the SBBAuthManagerDelegate instance (SBAUser) can call it when it gets a delegate callback from BridgeSDK.